frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -47,14 +47,16 @@ pods:
         file: new_user_password
     {{/TASKCFG_ALL_PASSWORD_SECRET_PATH}}
     {{/TASKCFG_ALL_SECURE_JMX_ENABLED}}
-    volume:
-      path: container-path
-      type: {{CASSANDRA_DISK_TYPE}}
-      size: {{CASSANDRA_DISK_MB}}
-      {{#CASSANDRA_VOLUME_PROFILE}}
-      profiles:
-        - {{CASSANDRA_VOLUME_PROFILE}}
-      {{/CASSANDRA_VOLUME_PROFILE}}
+    external-volumes:
```

Comment:
We should expose `driver-options` and `volume-name` to the user in the next patch series.